src/main/java/com/google/api/codegen/config/FlatteningConfig.java

```diff
@@ -305,7 +305,7 @@ private static FlatteningConfig createFlatteningFromProtoFile(
       }
 
       ResourceNameTreatment resourceNameTreatment = ResourceNameTreatment.NONE;
-      String resourceNameType = protoParser.getResourceType(parameterField.getProtoField());
```

Comment:
This is just a renaming of the function getResourceType() to getResourceReference()